unit-tests/func/frames/color_frame_frops_test.py

```diff
@@ -0,0 +1,52 @@
+from __future__ import division
```

Comment:
Please check if this import is needed - if so, consider adding a comment that explains why